### PR TITLE
272 fix hb streamers, added loader and added ellipsis for game

### PR DESCRIPTION
**Pull request by ronaldpabilonia**

### CODE REVIEW

File: js/streamers.js [hunk 1]
```diff
@@ -58,6 +58,7 @@
             if (typeof streamers.online === 'undefined') {
                 return null;
             }
+            $('#loader_img').attr('style', 'display: none');
```
Comment (nhvu1988) [inline js/streamers.js]: want to change all styles for this element?

Comment (ronaldpabilonia) [inline js/streamers.js]: it only has one style. it's for showing and hiding the loader. no other styles exist.

Comment (nhvu1988) [inline js/streamers.js]: but I think you should not do that, just for the future if anyone try to edit that element's styles :)

Comment (ronaldpabilonia) [inline js/streamers.js]: so what do you think is the better way to show and hide the loader?

Comment (nhvu1988) [inline js/streamers.js]:
```js
util.$('#loader_img').style.display = 'none';
```

Comment (ronaldpabilonia) [inline js/streamers.js]: huh? what do you mean util? this is heartbeattm, not heartbeat-comments-extension. im using jquery.

Comment (nhvu1988) [inline js/streamers.js]: you can use [css](http://api.jquery.com/css/) or [show](http://api.jquery.com/show/)/[hide](http://api.jquery.com/hide/) function

**Pull request merged**